files/qsub.pl

```diff
@@ -314,7 +314,7 @@ sub make_command
                 push(@command, "-e", $err_path);
             } else {
                 # jobname will be forced
-                my $jobbasename = $job_name ? basename($job_name) : "%j";
+                my $jobbasename = $job_name ? basename($job_name) : "%x";
```

Comment:
I think we changed this to %j at some point because of reasons :)

Reply:
@itkovian And do you remember what was the reason?

Reply:
Does "%x" exist if $job_name is not set?

Reply:
@itkovian Yes:
```
[vsc43020@gligar04 ~]$ ls -ltr 
total 261
drwxrwxr-x 2 vsc43020 vsc43020  4096 Feb  8 13:51 Desktop
drwxrwxr-x 3 vsc43020 vsc43020  4096 Feb  8 13:55 intel
drwxrwxr-x 2 vsc43020 vsc43020  4096 Feb 12 16:57 out
-rw-rw-r-- 1 vsc43020 vsc43020   203 Feb 13 08:49 test.sh
-rwxr-xr-x 1 vsc43020 vsc43020 35750 Feb 13 09:36 qsub.debug
-rwxr-xr-x 1 vsc43020 vsc43020 35172 Feb 13 09:39 qsub
-rw-rw-r-- 1 vsc43020 vsc43020    23 Feb 17 16:37 test2.sh
[vsc43020@gligar04 ~]$ cat test2.sh 
#!/bin/bash
echo Done!
[vsc43020@gligar04 ~]$ /usr/bin/sbatch -o /kyukon/home/gent/430/vsc43020/%x.o%A --export=NONE --get-user-env=60L test2.sh 
Submitted batch job 7754481 on cluster victini
[vsc43020@gligar04 ~]$ ls -ltr 
total 262
drwxrwxr-x 2 vsc43020 vsc43020  4096 Feb  8 13:51 Desktop
drwxrwxr-x 3 vsc43020 vsc43020  4096 Feb  8 13:55 intel
drwxrwxr-x 2 vsc43020 vsc43020  4096 Feb 12 16:57 out
-rw-rw-r-- 1 vsc43020 vsc43020   203 Feb 13 08:49 test.sh
-rwxr-xr-x 1 vsc43020 vsc43020 35750 Feb 13 09:36 qsub.debug
-rwxr-xr-x 1 vsc43020 vsc43020 35172 Feb 13 09:39 qsub
-rw-rw-r-- 1 vsc43020 vsc43020    23 Feb 17 16:37 test2.sh
-rw-rw-r-- 1 vsc43020 vsc43020     6 Feb 17 16:41 test2.sh.o7754481
[vsc43020@gligar04 ~]$ cat test2.sh.o7754481 
Done!
```